### ISSUE TITLE
Things missing from the API docs autosummary

### ISSUE THREAD

**Issue by kushalkolar** [pygfx/pygfx#872]
I just noticed that not everything that is available at the top level is shown in the public API page: https://github.com/pygfx/pygfx/blob/main/docs/reference.rst

Example: `PointsMarkerMaterial` and `PointsSpriteMaterial` are missing, they are accessible after going into thge `pygfx.Materials` doc page.

Idea: script that generates this page by grabbing all the top level cameras, controllers, geometries, materials etc. ?

**Comment by almarklein** [comment]
Yeah, manually keeping this list up-to-date  is a pain and likely to be forgotten. We have similar lists in the `__init__.py` of the subpackages. A script might be a good step to at least solve the problem for the total namespace. Or a test that just checks whether the list is complete.
